diag_sockets onversion completed

sbeaver-netgate opened the pull request.

sbeaver-netgate committed:
Conversion completed

sbeaver-netgate committed:
diag_sockets

Updated per SvL suggestions. - Thank you!

sbeaver-netgate committed:
Removed unneeded class and fixed tabs/spaces

SjonHortensius commented on usr/local/www/diag_sockets.php:
```diff
@@ -81,61 +77,78 @@
+						if (trim($line) == "")
+							continue;
+
+						print("<tr id=\"$name$i\">\n");
```
Both this _id_ and the _$class_ defined before it are not needed. There is no css or js that targets it

SjonHortensius commented on usr/local/www/diag_sockets.php:
```diff
@@ -81,61 +77,78 @@
+
+		<div class="table table-responsive">
+
+			<table class="table table-hover table-striped table-condensed" id="<?=$name?>">
```
This id isn't used anywhere. Could you also remove the redundant newlines around the `div`?

sbeaver-netgate committed:
dig_sockets updated as requested

Also fixed an incorrect </div> placement
I have retained the “$class = 'info’;” for now as I believe it
significantly improves the readability of the page. We can always
remove it if people don’t like it.
